docs/sources/configuration/_index.md

```diff
@@ -1801,8 +1843,9 @@ fifocache:
   [max_size_items: <int> | default = 0]
 
   # The expiry duration for the cache.
+  # The default value of 0 disables expiration.
   # CLI flag: -<prefix>.fifocache.duration
-  [validity: <duration> | default = 1h]
+  [validity: <duration> | default = 0s]
```

Comment:
same point here, this will default to `default_validity` if `default_validity` is non-zero. It will also gets modified when using boltdb shipper. In the case that the ingester or write target is being run, this ends up being set to 1 minute less than the `storage_config`'s index cache validity. I don't think we should include a default value at all here.